File: Day11_Max and MIn from array.py
def minmax(list):
    max = list[0]
    min = list[0]
    for i in range(1,len(list)):
        if list[i] > max:
            max = list[i]

        if list[i] < min:
            min  = list[i]

    return min, max


mylist = []
size = int(input("Enter the size of the list: "))
for i in range(1, size+1):
    element = int(input(f"{i}----->"))
    mylist.append(element)

min, max = minmax(mylist)
print("Min is: ",min, "Max is: ", max)

File: Day11_Max and MIn from array.cpp
#include<iostream>
using namespace std; 

void maxmin(int size, int array[]){
	int max, min;
	for(int i=0;i<size; i++){
		if(array[i] > max){
			 max = array[i];
		}
		if(array[i] < min){
			 min = array[i];
		}
	}
	cout<<"Max: "<<max<<"& Min :"<<min;

}

int main(){
	
	int n;
	cout<<"Enter the size of the array: ";
	cin>>n;
	int array[n];
	cout<<"Enter an array of size "<<endl;
	for(int i=0; i<n; i++){
		cin>>array[i];
	}
	
	maxmin(n, array);
	
	return 0;
}
